### PR TITLE
optional highlight of a country to enhance visibility

### PR DESCRIPTION
**Pull request by hweigel**
Hi @bibaldo 

Cosmetic enhancement, allows for one row configurable by country name to have a different background, thus enhancing visibility.

I think this time I got it right, even managed to update README.md without reminder :-D

Stay safe and healthy
/H

### CODE REVIEW

File: MMM-COVID19.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ Module.register("MMM-COVID19", {
     worldStats: false,
     delta: false,
     showExtraInfo: false,
+	highlightCountry: "", // when containing a valid country ("countries:...") the row's background colour will be changed to enhance visibility
```
Comment (bibaldo) [inline MMM-COVID19.js]: @hweigel 2 white spaces missing

File: MMM-COVID19.js [hunk 2]
```diff
@@ -232,6 +233,9 @@ Module.register("MMM-COVID19", {
 
         countryNameCell.innerHTML = this.translate(countryName)
         countryNameCell.className = this.config.infoRowClass
+		if (countryName === this.config.highlightCountry) {
```
Comment (bibaldo) [inline MMM-COVID19.js]: @hweigel please indent it properly

Comment (hweigel) [reply MMM-COVID19.js]: ...and done (I hope)

File: MMM-COVID19.js (hunk 1 above)
Comment (hweigel) [reply MMM-COVID19.js]: done...

**Comment by hweigel** [comment]
Disclaimer: I **suck** at coding :-)

Meaning I have to apologise, that's notepad++ messing it up for me.

I'm sure there's a trick to configure notepad++ for github code, but I have yet to figure that one out.

**Comment by bibaldo** [comment]
> Disclaimer: I **suck** at coding :-)
> 
> Meaning I have to apologise, that's notepad++ messing it up for me.
> 
> I'm sure there's a trick to configure notepad++ for github code, but I have yet to figure that one out.

naaa, don't worry. Your contributions are very good.

The Notepad thing is easy to solve. Just go to preferences and change how you are indenting (whitespaces over tabs and 2 whitespaces)

File: MMM-COVID19.js [hunk 3]
```diff
@@ -232,6 +233,9 @@ Module.register("MMM-COVID19", {
 
         countryNameCell.innerHTML = this.translate(countryName)
         countryNameCell.className = this.config.infoRowClass
+	      if (countryName === this.config.highlightCountry) {
```
Comment (bibaldo) [inline MMM-COVID19.js]: sorry but you have here 6 extra whitespaces

Comment (hweigel) [reply MMM-COVID19.js]: Darn, shoulda seen it myself. Fixed. Thanks for being patient with me!

Comment (bibaldo) [reply MMM-COVID19.js]: @hweigel sorry but I still see it wrong (indentation)

Comment (hweigel) [reply MMM-COVID19.js]: @bibaldo I see it here in the change browser too, but when I open the change, it shows the correct indentation:

Clipboard-3

Comment (bibaldo) [reply MMM-COVID19.js]: don't worry, we will merge it anyway and I will fix it on master.

Sorry for the inconveniences

**Comment by hweigel** [comment]
Hi Jose,
one last question: was the intendation ok after merging? 

Warm Regards
Helmut

**Pull request merged**